analyzer/src/main/kotlin/managers/SpdxDocumentFile.kt

```diff
@@ -271,6 +271,23 @@ private fun String.isExternalDocumentReferenceId(): Boolean = startsWith(SpdxCon
  */
 private fun String.mapNotPresentToEmpty(): String = takeUnless { SpdxConstants.isNotPresent(it) }.orEmpty()
 
+/**
+ * Wrap any "present" SPDX value in a sorted set, or return an empty sorted set otherwise.
+ */
+private fun String?.wrapPresentInSortedSet(): SortedSet<String> {
+    if (this == null || SpdxConstants.isNotPresent(this)) return sortedSetOf()
+
+    val persons = withoutPrefix(SpdxConstants.PERSON)?.let { persons ->
+        // In case of a person, allow a comma-separated list of persons.
+        persons.split(',').mapTo(sortedSetOf()) { it.trim() }
+    }
+
+    // Do not split an organization like "Acme, Inc." by comma.
+    val organization = withoutPrefix(SpdxConstants.ORGANIZATION)?.let { sortedSetOf(it) }
```

Comment:
Should this use the `extractOrganization()` function? Accordings to the docs is supposed to be used for an "originator" string.

Reply:
Actually I decided to do somewhat the opposite and remove the `extractOrganization()` function. See the commit message for the rationale.